test_files/namespaced/export_namespace.js.transform.patch

```diff
@@ -0,0 +1,11 @@
+Index: ./test_files/namespaced/export_namespace.js
+===================================================================
+--- ./test_files/namespaced/export_namespace.js	golden
++++ ./test_files/namespaced/export_namespace.js	tsickle with transformer
+@@ -17,5 +17,5 @@
+     function Interface() { }
+     typeNamespace.Interface = Interface;
+     function Interface_tsickle_Closure_declarations() {
+     }
+-})(typeNamespace = exports.typeNamespace || (exports.typeNamespace = {}));
```

Comment:
This appears to be a bug in the emit with transformers and namespaces. I'll file an issue.